src/NumberField/NumberField.stories.tsx

```diff
@@ -157,8 +164,31 @@ export const InitialError: Story = {
     helperText: "Oh no you've made a mistake!",
     label: "Enter a Number",
     margin: "normal",
+    max: 1,
```

Comment:
You've set the initial value on this story to something that shows an error message other than the one passed in. Suggest that you set the initial value to something valid / don't include a min/,ax for this story.